widget_driver/pubspec.yaml

```diff
@@ -19,5 +19,6 @@ dev_dependencies:
     sdk: flutter
   flutter_lints: ^1.0.4
   mocktail: ^0.1.4
+  provider: ^6.0.4
```

Comment:
Do you also want to bump the version in here.. so that we can deploy a new version of the pacakge?